index.html

```diff
@@ -1189,7 +1190,9 @@
       interface NDEFRecord {
         readonly attribute NDEFRecordType recordType;
         readonly attribute USVString mediaType;
-        NDEFRecordData data();
+        [NewObject] USVString toText();
+        [NewObject] ArrayBuffer toArrayBuffer();
+        [NewObject] any toJSON();
```

Comment:
Can you use `[Default] object toJSON();`?  If no... can you define a dictionary? 

```suggestion
        [NewObject] object toJSON();
```

Reply:
Also, there is a IDL `record` type... NDEFRecord being a "Record" seems like a there might be overlap? Or maybe "Record" is a misnomer?

Reply:
> Can you use [Default] object toJSON();? If no... can you define a dictionary?

No, I don't think so, as it only try to convert the bytes of the `NDEFRecord` interface to `JSON` object. And the bytes is even just a concept of the `NDEFRecord` interface.